### PR TITLE
bug(detail): submit comment

### LINKED ISSUE #116: feat(recipe-details): add recipe details comments (SMALL)

### PR DESCRIPTION
**Pull request by FangchengDou**
add submit img feature but I don't know how to use onchange=""call the function,  error isthis.uploadImg is not a function. This bug also exists on contribute page. There are lots of style bugs or syntax errors on this page. The best way is to rewrite it. What do you think? If the bad syntax (eg. all elements are class) doesn't obstruct next step development, I will just correct the style, if it obstruct, I will rewrite.
close #116

### CODE REVIEW

File: source/components/recipe-details/recipe-details.html [hunk 1]
```diff
@@ -105,14 +105,23 @@
...
-User input here</textarea
-      >
+    <form class="comment-box-input">
+      <textarea
+        rows="4"
+        cols="50"
+        style="font-size: 18pt"
+        placeholder="User input here"
+      ></textarea>
+      <input type="file" id="submit-img" onchange="this.uploadImg(event)" />
```
Comment (jbutera94) [inline source/components/recipe-details/recipe-details.html:115]: I don't see the file input. Are users supposed to be able to upload images in comments?

Also I think for the `uploadImg` function, using `onchange="uploadImg(event)"` should work. Let me know if it doesn't.

**Review by jbutera94 — CHANGES_REQUESTED** [review]
Looks good to me, left one suggestion. For the syntax errors, I think it's good enough for now. I wouldn't spend too much time perfecting it, and instead would continue iterating the design with Viki. Also don't worry about the errors in Codacy, it's not completely set up yet.

File: source/components/recipe-details/recipe-details.html (hunk 1 above)
Comment (FangchengDou) [reply source/components/recipe-details/recipe-details.html:115]: Viki says there should be a upload img feature when comment. But if you think it not good, I can remove it. When you click on the image, you should be able to submit the img. Same with contribute page. But this function is doesn't work also on contribute page.

Comment (jbutera94) [reply source/components/recipe-details/recipe-details.html:115]: Hmm... I'm not sure why that wouldn't work (also sorry, I just saw this comment).

Comment (jbutera94) [reply source/components/recipe-details/recipe-details.html:115]: @FangchengDou Let's focus on the other pages for now. We can figure out why this isn't working once we have the CRUD pages improved.

**Pull request closed without merging**